src/partition/repart.c

```diff
@@ -375,6 +399,37 @@ static int context_add_free_area(
         return 0;
 }
 
+static Partition* find_verity_sibling(Partition *head, Partition *p) {
```

Comment:
hmm, once we add the pkcs7 signature stuff there will not be a single sibling anymore...

maybe add a VerityMode parameter, and just search for that type? sounds more future proof